static/panes/editor.js

```diff
@@ -408,7 +446,7 @@ Editor.prototype.disableVim = function () {
 Editor.prototype.initButtons = function (state) {
     this.fontScale = new FontScale(this.domRoot, state, this.editor);
     this.languageBtn = this.domRoot.find('.change-language');
-    // Ensure that the button is disabled if we don't have nothing to select
+    // Ensure that the button is disabled if we don't have anything to select
```

Comment:
Oops. This was the only change in editor, but prettier stepped in in the pre-commit hook and changed the whole file